design/sketch.webidl

```diff
@@ -501,12 +501,19 @@ dictionary WebGPUBufferCopyView {
     u32 imageHeight;
 };
 
+typedef u32 WebGPUTextureAspectFlags;
+interface WebGPUTextureAspect {
```

Comment:
I assumed this would be an enum instead of a bitfield because of the following line in the vulkan spec for the validation of `VkBufferImageCopy`:

> The aspectMask member of imageSubresource must only have a single bit set

That said it would be ok to allow both aspects to be copied at once and split the copy on the Vulkan backend

Reply:
FYI, Vulkan doesn't require an implementation to support buffer-to-texture (or texture-to-texture for the matter) copies with depth-stencil involved. See [https://vulkan.lunarg.com/doc/view/1.0.30.0/linux/vkspec.chunked/ch31s03.html](Table 31.19. Mandatory format support: 64-bit/uneven channels and depth/stencil).

I don't think we want to support copying depth/stencil *separately*, although I'd be open to discuss this :)
Maybe this aspect flag isn't needed here at all then?

Reply:
The aspect flag is also used for texture views and sampling. It's how you choose between sampling the depth and stencil components of a DS texture.

Reply:
Sure, and for texture views it doesn't make sense for it to be an enum. I suggest to leave it as bitflags and remove the use of it from `WebGPUTextureCopyView`